Description: Change default programs
 x-terminal-emulator is the best default for a terminal emulator on
 Debian, and xlock is not available at all.
Forwarded: not-needed
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2016-07-16
Index: spectrwm/spectrwm.c
===================================================================
--- spectrwm.orig/spectrwm.c
+++ spectrwm/spectrwm.c
@@ -8109,10 +8109,10 @@ validate_spawns(void)
 void
 setup_spawn(void)
 {
-	setconfspawn("lock",		"xlock",		0);
+	setconfspawn("lock",		"slock",		0);
 
-	setconfspawn("term",		"xterm",		0);
-	setconfspawn("spawn_term",	"xterm",		0);
+	setconfspawn("term",		"x-terminal-emulator",	0);
+	setconfspawn("spawn_term",	"x-terminal-emulator",	0);
 
 	setconfspawn("menu",		"dmenu_run"
 					" $dmenu_bottom"
Index: spectrwm/spectrwm.1
===================================================================
--- spectrwm.orig/spectrwm.1
+++ spectrwm/spectrwm.1
@@ -609,9 +609,9 @@ bind[] = MOD+Shift+b
 Default programs:
 .Bl -tag -width "screenshot_wind" -offset indent -compact
 .It Cm term
-xterm
+x\-terminal\-emulator
 .It Cm lock
-xlock
+slock
 .It Cm menu
 dmenu_run $dmenu_bottom \-fn $bar_font \-nb $bar_color \-nf $bar_font_color
 \-sb $bar_color_selected \-sf $bar_font_color_selected
